hack/scan-image-vuln.sh

```diff
@@ -78,7 +78,7 @@ if util::cmd_exist trivy ; then
 	echo "pass"
 else
 	echo "start installing trivy"
-	curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b /usr/local/bin v0.48.1
+	curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b /usr/local/bin v0.53.0
```

Comment:
```suggestion
	curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b /usr/local/bin v0.56.1
```
`trivy-action` has supported `trivy v0.56.1` starting from version `v0.25.0`. Please update it, and then LGTM~

Reply:
@zhzhuang-zju rectified